### PR TITLE
(Publish 18 March 2024) lambdacommunityplus

### PR DESCRIPTION
**Pull request by lchockalingam**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

Add a what's new post for Lambda extension which is in community plus tag and maintained and improved by New Relic engineers

### CODE REVIEW

**Comment by lchockalingam** [comment]
no PMM review needed

File: src/content/whats-new/2024/03/whats-new-03-18-lambdacommunityplus.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+The lightweight New Relic AWS Lambda extension runs alongside your AWS Lambda functions and allows you to collect, enhance, and transport telemetry data from your AWS Lambda functions to New Relic automatically, without requiring an external transport such as CloudWatch Logs or Kinesis. 
+
+Did you know that the New Relic AWS Lambda extension follows a “Community plus” approach? This means it benefits from a collaborative development process with active community involvement. 
+
+Let’s dive into what this entails for you:
+
+* **Active maintainers:** A dedicated team of engineers within New Relic maintains this project, ensures the extension's stability and continuous improvement.
+* **Open development:** The code is on GitHub, welcoming your contributions through issues and pull requests. 
+* **Community support:** The New Relic Explorers Hub, a vibrant community forum, is your one-stop shop for troubleshooting with fellow users and enthusiasts. If you are
```
Comment (rhetoric101) [inline src/content/whats-new/2024/03/whats-new-03-18-lambdacommunityplus.md:17]:
```suggestion
* **Community support:** The New Relic Explorers Hub, a vibrant community forum, is your one-stop shop for troubleshooting with fellow users and enthusiasts.
```

File: src/content/whats-new/2024/03/whats-new-03-18-lambdacommunityplus.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+getStartedLink: 'https://github.com/newrelic/newrelic-lambda-extension'
+---
+
+The lightweight New Relic AWS Lambda extension runs alongside your AWS Lambda functions and allows you to collect, enhance, and transport telemetry data from your AWS Lambda functions to New Relic automatically, without requiring an external transport such as CloudWatch Logs or Kinesis. 
+
+Did you know that the New Relic AWS Lambda extension follows a “Community plus” approach? This means it benefits from a collaborative development process with active community involvement. 
+
+Let’s dive into what this entails for you:
+
+* **Active maintainers:** A dedicated team of engineers within New Relic maintains this project, ensures the extension's stability and continuous improvement.
```
Comment (rhetoric101) [inline src/content/whats-new/2024/03/whats-new-03-18-lambdacommunityplus.md:15]:
```suggestion
* **Active maintainers:** A dedicated team of engineers within New Relic maintains this project, ensures the extension's stability, and provides continuous improvement.
```

**Review by rhetoric101 — COMMENTED** [review]
@lchockalingam thanks for adding this post! I just made a few minor suggestions below. Could you take a look?

**Review by homelessbirds — APPROVED** [review]

**Pull request merged**